Dockerfile

```diff
@@ -4,21 +4,25 @@
 
 FROM docker.io/library/golang:bookworm AS build
```

Comment:
Do we want to use a different image (following today's discussion)

Reply:
Meh. It can wait.